src/BackgroundCells.js

```diff
@@ -151,12 +169,18 @@ class BackgroundCells extends React.Component {
         if (pointInBox(rowBox, point)) {
           let width = slotWidth(getBoundsForNode(node), range.length);
           let currentCell = getCellAtX(rowBox, point.x, width, rtl, range.length);
-
-          this._selectSlot({
-            startIdx: currentCell,
-            endIdx: currentCell,
-            action: 'click',
-          });
+          const cellId = node.childNodes
+            .item(currentCell)
+            .querySelector('.rbc-day-bg')
+            .getAttribute('id');
+          const date = this.parseDateFromId(cellId);
+          this.props.onNavigate(null, date);
+
+          // this._selectSlot({
```

Comment:
Commenting this code as is causing selection duplicates while doing a click and then pressing arrow keys.